flexmeasures/utils/time_utils.py

```diff
@@ -333,3 +335,15 @@ def duration_isoformat(duration: timedelta):
     # at least one component has to be there.
     repl = ret and "".join(ret) or "T0H"
     return re.sub("%P", repl, "P%P")
+
+
+def determine_minimum_resampling_resolution(
```

Comment:
In my PR in flexmeasures-simulate, I have a similar util function, and I decided to let it `raise` if the resolutions are actually not compatible (`if any([resolution % min_resolution != timedelta(hours=0) for resolution in resolutions]`)

This function could do this, maybe triggered by a parameter flag `raise_if_incompatible`. Thoughts?

Reply:
In my mind, no combination of resolutions should be incompatible. Resampling a (simplified) time series with instantaneous values should be rather straightforward: when downsampling, select the first value; when upsampling, interpolate using some strategy (such as forward fill or linear interpolation), which we could later define as a sensor attribute. I'll address this soon in a timely-beliefs PR.